easybuild/easyblocks/n/neuron.py

```diff
@@ -190,6 +240,10 @@ def sanity_check_step(self):
         else:
             self.log.info("Skipping MPI testing of NEURON since MPI testing is disabled")
 
+        if self.with_python:
+            cmd = "python -c 'import neuron; neuron.test()'"
+            (out, ec) = run_cmd(cmd, simple=False, log_all=True, log_output=True)
```

Comment:
for the `sanity_check_step`, doing above `neuron.test()` is quite good test.

IMO, we don't need to check all binaries and libs mentioned above (as they might change with versions). Important ones are : neurondemo, nrniv, lib, python site-packages.

Reply:
@boegel, thoughts on this ?

Reply:
Just an additional note about `neuron.test()` : this will run NEURON. In HPC environment, MPI stuff will be linked and that might cause problem (depending on the platform and where this build is running). A **minimal** check would be `from neuron import h`.

Reply:
@mboisson Cleaning up the sanity check to only require a subset of files makes sense, especially if it helps to avoid having to update the easyblock when things change over time.

W.r.t. the MPI test: good point, maybe we should make that part easy to skip in case of trouble?

Reply:
well, we already have the test to skip MPI testing, no ? 
line 217.

Reply:
@pramodk, aren't `nrnmpi`, `nrnpython`  key as well ?

Reply:
>  aren't nrnmpi, nrnpython key as well ?

@mboisson  : I didn't understand the question. What do you mean `key`?  do you mean MPI and Python is important to test?

Reply:
I mean they used to be libraries (libnrnmpi, libnrnpython) in previous versions, but I see they aren't there in 7.8.2, so I guess those are libraries that could be removed in different versions, and we probably shouldn't test for them ?

Reply:
Ah yes. With CMake we are no longer creating those extra libraries. Most of the stuff is now packaged under libnrniv.so. 

(Sorry for the trouble! After upcoming major release 8.0, most likely autotools will be removed and it would be then easy to clean the recipes)